### PR TITLE
Fix `GameWindow.IsActive` and on-screen keyboard for Android

### PR DESCRIPTION
**Pull request by hex11**
`GameWindow.IsActive` is always `false` on Android when `AndroidGameWindow` overrided `Focused` to be `true`.

https://github.com/ppy/osu-framework/blob/dd8d1136cb386aea8b7a6fe5db5af6dfd3458fb9/osu.Framework.Android/AndroidGameWindow.cs#L19

Overriding `Focused` property doesn't work for Android but do work for iOS:

https://github.com/ppy/osu-framework/blob/dd8d1136cb386aea8b7a6fe5db5af6dfd3458fb9/osu.Framework.iOS/IOSGameWindow.cs#L30

I guess it's because some events (used to update `IsActive`) never get raised on Android, like this one:

https://github.com/ppy/osu-framework/blob/dd8d1136cb386aea8b7a6fe5db5af6dfd3458fb9/osu.Framework/Platform/GameWindow.cs#L86

So I just make `IsActive` default to true on Android.

### CODE REVIEW

File: osu.Framework.Android/AndroidGameWindow.cs [hunk 1]
```diff
@@ -32,6 +32,8 @@ public override void SetupWindow(FrameworkConfigManager config)
         {
             // Let's just say the cursor is always in the window.
             CursorInWindow = true;
+            // And the window is always active.
+            SetActive(true);
```
Comment (smoogipoo) [inline osu.Framework.Android/AndroidGameWindow.cs]: It's unnecessary to create a method that does this, override `Focused` and force it to `true` instead

Comment (hex11) [reply osu.Framework.Android/AndroidGameWindow.cs]: Already overrided but doesn't work.

Comment (hex11) [reply osu.Framework.Android/AndroidGameWindow.cs]: Maybe we should just add `isActive.Value = Focused;` in constructor of GameWindow?

Comment (smoogipoo) [reply osu.Framework.Android/AndroidGameWindow.cs]: https://github.com/ppy/osu-framework/blob/master/osu.Framework/Platform/GameWindow.cs#L95

It's already there, albeit on the first update. Is this somehow too late for you?

Comment (hex11) [reply osu.Framework.Android/AndroidGameWindow.cs]: But `IsActive` never get updated on Android and I don't know why, as described in PR.

Comment (hex11) [reply osu.Framework.Android/AndroidGameWindow.cs]: (Is it possible that `UpdateFrame` event never get raised?)

Comment (tangalbert919) [reply osu.Framework.Android/AndroidGameWindow.cs]: This method is definitely unnecessary if only Android is going to use it. You must find another way to fix this problem.

**Comment by swoolcock** [comment]
Resolved by:
https://github.com/ppy/osuTK/pull/50
https://github.com/ppy/osu-framework/pull/2454

File: osu.Framework.Android/AndroidGameWindow.cs (hunk 1 above)
Comment (SilkyPantsDan) [reply osu.Framework.Android/AndroidGameWindow.cs]: The issue with UpdateFrame never being called was due to the View not being run, but the window instead - this has been addressed in PR https://github.com/ppy/osu-framework/pull/2454.

**Pull request closed without merging**